lms/static/styles/components/_ErrorDisplay.scss

```diff
@@ -2,13 +2,23 @@
 
 
 .ErrorDisplay {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  overflow-y: scroll;
+  padding-right: 5px;
```

Comment:
Curious about this unbalanced right padding...can you help me understand? (i.e. why is there no left padding?)

Reply:
awe yes, I think that was so it didn't touch the scroll bar and look too squished, I'll double check.